app/assets/javascripts/widgets/hero_panel.js

```diff
@@ -0,0 +1,10 @@
+$(document).ready(function(){
+  var nav = $('.program-select');
```

Comment:
Can you make this select a little more specifc?  Why don't you add an ID to the program select called `dropdown-select-links` then select that in the Javascript.